src/main/webapp/app/exam/exam-scores/exam-score-dtos.model.ts

```diff
@@ -25,6 +27,7 @@ export class ExerciseInfo {
     public title: string;
     public maxPoints: number;
     public numberOfParticipants: number;
+    public exerciseType: string;
```

Comment:
Don't we have an enum type that represents the server-side enums?

Reply:
We do not have a server-side exercise type enum. I saw us using the actual Class name of the exercise which is then TextExercise for example, which is was I am doing in this case